src/SplitPackages/Program.cs

```diff
@@ -159,10 +160,49 @@ private int Execute()
             }
         }
 
-        private void CreateProjectJsonFileForOptimizedCache(string destinationPath, ClassificationResult optimizedCacheClassification)
+        private void CreateProjectJsonFilesForOptimizedCache(string destinationPath, ClassificationResult optimizedCacheClassification)
+        {
+            CreateProjectJsonFileWithTimeStamps(destinationPath, optimizedCacheClassification);
+            CreateProjectJsonFileWithoutTimeStamps(destinationPath, optimizedCacheClassification);
+        }
+
+        private void CreateProjectJsonFileWithTimeStamps(string destinationPath, ClassificationResult optimizedCacheClassification)
+        {
+            var packages = optimizedCacheClassification.GetPackagesForValue("include");
+            var builder = CreateBaseJsonFileBuilderForCache(destinationPath, "cache.project.json");
+            builder.AddDependencies(packages);
+            builder.Execute();
+        }
+
+        private void CreateProjectJsonFileWithoutTimeStamps(string destinationPath, ClassificationResult optimizedCacheClassification)
+        {
+            var packages = optimizedCacheClassification.GetPackagesForValue("include");
+            var correctedPackages = packages.Select(p => new PackageInformation(p.FullPath, p.Identity, GetFinalVersion(p.Version), p.SupportedFrameworks));
+
+            var builder = CreateBaseJsonFileBuilderForCache(destinationPath, "final.cache.project.json");
+            builder.AddDependencies(correctedPackages);
+            builder.Execute();
+        }
+
+        private string GetFinalVersion(string version)
+        {
+            var frameworkVersion = new NuGetVersion(version);
+            if (frameworkVersion.Release.Contains("rtm"))
```

Comment:
Could we do `StartsWith("rtm-")`? Just so that it's consistent with Coherence-Signed? Ok to do this in 1.0.1 if it works fine now.